Classes/UT3Nightshade.uc

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright � 2012 100GPing100
- * Copyright � 2014 GreatEmerald
+ * Copyright � 2012, 2017, 2018 Luís 'zeluisping' Guimarães (100GPing100)
+ * Copyright � 2014 GreatEmerald

```

Comment:
The copyright symbols got messed up. Make sure all files are in UTF-8 encoding.

Reply:
This is weird because that's exactly how it shows in my editor and encoding is set to UTF-8, I even changed to it manually. Gotta figure out what's happening here, thanks for pointing this out!